links/src/utils/getSibling.ts

```diff
@@ -0,0 +1,11 @@
+import { toTree, Tree, GetSiblings } from "./generateLinks";
+
+// getSiblings(tree, '/cancerqld/research')
```

Comment:
Make sure your comments are verbose and relevant [How to comment using JSDoc notation](https://devhints.io/jsdoc)